pom.xml

```diff
@@ -148,6 +153,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
I think the project purposefully tried to use an earlier version of Java. I seem to remember avoiding lambdas.

Is this an okay change? Might require a major bump. Not sure.

Reply:
The `File.walk` is only available since java 8. If bumping is out of option then we might want to close/rework this PR.

Is bumping to a major version an ok option here @malaporte ?